### PR TITLE
[10.0][MIG] stock_picking_delivery_rate: Migrate to 10.0

### PR DESCRIPTION
**Pull request by tedsalmon**
* Re-add `partner_id` to `delivery.carrier`

cc @lasley

### CODE REVIEW

File: stock_picking_delivery_rate/README.rst [hunk 1]
```diff
@@ -55,8 +58,7 @@ Contributors
 ------------
 
 * Dave Lasley <dave@laslabs.com>
-* Brett Wood <bwood@laslabs.com>
-* Ted Salmon <tsalmon@laslabs.com>
+
```
Comment (lasley) [inline stock_picking_delivery_rate/README.rst]: Why did you remove Brett and Your attributions?

File: stock_picking_delivery_rate/__manifest__.py [hunk 2]
```diff
@@ -1,15 +1,15 @@
 # -*- coding: utf-8 -*-
-# Copyright 2017 LasLabs Inc.
-# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+# Copyright 2016-2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (lasley) [inline stock_picking_delivery_rate/__manifest__.py]: LasLabs uses LGPL for all Odoo modules not depending on AGPL assets

File: stock_picking_delivery_rate/__manifest__.py [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 
 {
-    "name": "Stock Picking Delivery Rate",
+    "name": "Stock Picking Rate",
```
Comment (lasley) [inline stock_picking_delivery_rate/__manifest__.py]: I feel like you reverted a lot of things that shouldn't have been.

**Review by lasley — CHANGES_REQUESTED** [review]
@tedsalmon I did not finish this review. Please double check your migration commit

**Comment by tedsalmon** [comment]
@lasley,

I accidentally submitted the wrong migration branch that I had for this module. I'll go ahead and push up the commits from the correct branch into this one.

Thanks & Sorry!

File: stock_picking_delivery_rate/README.rst [hunk 4]
```diff
@@ -39,7 +39,7 @@ Bug Tracker
 ===========
 
 Bugs are tracked on `GitHub Issues
-<https://github.com/OCA/stock-logistics-workflow/issues>`_. In case of trouble, please
+`<https://github.com/OCA/carrier-delivery/issues>`_. In case of trouble, please
```
Comment (lasley) [inline stock_picking_delivery_rate/README.rst]: delivery-carrier

Also, looks like this is wrong on  OCA/9.0. Can you please submit a quick hotfix for that?

File: stock_picking_delivery_rate/models/__init__.py [hunk 5]
```diff
@@ -1,6 +1,7 @@
 # -*- coding: utf-8 -*-
-# Copyright 2017 LasLabs Inc.
-# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+# Copyright 2016-2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```
Comment (lasley) [inline stock_picking_delivery_rate/models/__init__.py]: Seriously duder you're killing me with the AGPL.

**Review by lasley — CHANGES_REQUESTED** [review]

**Review by lasley — APPROVED** [review]
Thanks @tedsalmon

**Review by woodbrettm — APPROVED** [review]
LGTM!
